File: src/test/ironfist/next/items/test/RingIdentityResourcesTest.java
package ironfist.next.items.test;

import java.text.MessageFormat;
import java.util.ResourceBundle;

import junit.framework.TestCase;

/**
 * DOCUMENT ME!
 * 
 * @author pmgremo
 */
public class RingIdentityResourcesTest extends TestCase {

  /**
   * Constructor for RingIdentityResourcesTest.
   * 
   * @param arg0
   */
  public RingIdentityResourcesTest(String arg0) {
    super(arg0);
  }

  public void testLoadResource() throws Exception {
    ResourceBundle bundle = ResourceBundle.getBundle("ironfist.next.items.RingIdentityResources");
    String pattern = bundle.getString("ring.protection.identity");
    System.out.println(MessageFormat.format(pattern, new Object[]{new Integer(
      -1)}));
    System.out.println(MessageFormat.format(pattern, new Object[]{
        new Integer(1),
        new Integer(-1)}));
    System.out.println(MessageFormat.format(pattern, new Object[]{
        new Integer(1),
        new Integer(+12)}));
    System.out.println(MessageFormat.format(pattern, new Object[]{
        new Integer(2),
        new Integer(0)}));
    System.out.println(MessageFormat.format(pattern, new Object[]{
        new Integer(3),
        new Integer(+2)}));
  }
}

File: src/test/java/ukalus/items/RingIdentityResourcesTest.kt
package ukalus.items

import java.text.MessageFormat
import java.util.ResourceBundle

import junit.framework.TestCase

/**
 * DOCUMENT ME!

 * @author pmgremo
 */
class RingIdentityResourcesTest : TestCase() {

    fun testLoadResource() {
        val bundle = ResourceBundle.getBundle("ukalus.items.RingIdentityResources")
        val pattern = bundle.getString("ring.protection.identity")
        println(MessageFormat.format(pattern, *arrayOf<Any>(-1)))
        println(MessageFormat.format(pattern, *arrayOf<Any>(1, -1)))
        println(MessageFormat.format(pattern, *arrayOf<Any>(1, +12)))
        println(MessageFormat.format(pattern, *arrayOf<Any>(2, 0)))
        println(MessageFormat.format(pattern, *arrayOf<Any>(3, +2)))
    }
}